spec/models/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'User model' do
+    before(:each) do
+      @user = User.new(name: 'Daniel', photo: 'photo_link', bio: 'my bio')
+    end
+
+    before { @user.save }
+
+    it 'name must not be blank' do
+      @user.name = nil
+      expect(@user).to_not be_valid
+    end
+
+    it 'post counter must be an integer' do
+      @user.posts_counter = 'hello'
+      expect(@user).to_not be_valid
+    end
+
+    it 'should have 0 posts' do
+      expect(@user.recent_posts.length).to eq 0
+    end
+  end
+end
```

Comment:
- [ ] You should add a test for the `recent_posts` method. Kindly add it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec_n%2B1.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.)